File: rcmfadmin/Models/Donor.cs
namespace rcmfadmin.Models;


public class Donor : IDbItem<int>
{
  public int Id { get; set; }
  public string Name { get; set; }
  public string Email { get; set; }
  public int? Amount { get; set; }
  public DateTime CreatedAt { get; set; }
  public DateTime UpdatedAt { get; set; }
}

File: rcmfadmin.client/src/models/Donor.js
export class Donor {
  constructor(data) {
    this.id = data.id;
    this.name = data.name
    this.email = data.email
    this.amount = data.amount
    this.createdAt = data.createdAt
    this.updatedAt = data.updatedAt
  }
}

